omnibus/files/private-chef-cookbooks/private-chef/recipes/elasticsearch.rb

```diff
@@ -159,7 +160,30 @@
 end
 
 component_runit_service 'elasticsearch' do
-  action [:enable, :start]
+  action :enable
+end
+
+# start the elasticsearch runit service.
+ruby_block 'starting elasticsearch' do
+  block do
+    started = false
+    10.times do
```

Comment:
Looks like there is an option to use retries and retry_delay instead of x.times and sleep: https://docs.chef.io/resource_common/#properties